tests/end_to_end.rs

```diff
@@ -1,3 +1,5 @@
+mod http;
+
```

Comment:
Is this actually needed? The tests in `http.rs` ran fine in my case without it.

Reply:
Not quite sure; CLion was complaining that `File is not included in the module tree, analysis is not available` and the intellisense died, until I added it here.